examples/asa/scripts/transfer/common.js

```diff
@@ -9,11 +9,30 @@ exports.transferAlgo = async function (deployer, senderAddr, receiverAddr, amoun
     }
 }   
 
-exports.transferASA = async function (deployer, senderAccount, receiverAddr, amount, assetID, lsig){
+exports.transferASA = async function (deployer, senderAccount, receiverAddr, amount, assetID, lsig) {
     try {
         const details = await transferASALsig(deployer, senderAccount, receiverAddr, amount, assetID, lsig);
         console.log(details);
     } catch (e) {
         console.error('Transaction Failed', e.response.error);
     }
 }
+
+exports.transferAlgoAtomic = async function (deployer, txnParams) {
```

Comment:
Let's rename this function and add `Micro` to the name: 

    transferMicroAlgoAtomic

Reply:
ok